.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+
+on:
+  pull_request:
+    branches:
+      - mc-develop
+  push:
+    branches:
+      - mc-develop
+
+env:
+  RUST_BACKTRACE: full
+  CARGO_INCREMENTAL: 0
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly-2022-04-29
```

Comment:
Consider adding a `rust-toolchain` file for consistency across local and CI runs. `actions-rs/toolchain` honors that file.

Reply:
There is one in the branch already.